tools/kafka/loopback/bin/entrypoint.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+
+# -e       Exit immediately if a command exits with a non-zero status.
+# -x       Print commands and their arguments as they are executed
+set -e
+
+# DEBUG
+if [ ! -z "${DEBUG+x}" ]; then
+    set -x
+fi
+
+
+echo "Loopback start"
+
+DOJOT_USERNAME=${DOJOT_USERNAME:-"admin"}
+DOJOT_PASSWORD=${DOJOT_PASSWORD:-"admin"}
+
+AUTH_ADDRESS=${AUTH_ADDRESS:-"http://auth:5000"}
+DATA_BROKER_ADDRESS=${DATA_BROKER_ADDRESS:-"http://data-broker:80"}
+KAFKA_BROKER_LIST=${KAFKA_BROKER_LIST:-"kafka-server:9092"}
+
+LOOPBACK_CONSUMER_GROUP=${LOOPBACK_CONSUMER_GROUP:-"loopback-group"}
+DEVICE_DATA_TOPIC=${DEVICE_DATA_TOPIC:-"device-data"}
+DEVICE_MANAGER_TOPIC=${DEVICE_MANAGER_TOPIC:-"dojot.device-manager.device"}
+
+# auth
+readonly AUTH_DATA="{\"username\": \"${DOJOT_USERNAME}\", \"passwd\":\"${DOJOT_PASSWORD}\"}"
+readonly JSON_CONTENT_TYPE="Content-Type:application/json"
+readonly TOKEN=$(curl --silent -X POST ${AUTH_ADDRESS} -H "${JSON_CONTENT_TYPE}" -d "${AUTH_DATA}" | jq '.jwt' -r)
+
+if [ ! -z "$TOKEN" ]
+then
+    echo "Retrived token sucessfully ..."
+    LDEVICE_DATA_TOPIC=$(curl --silent -X GET ${DATA_BROKER_ADDRESS}/topic/${DEVICE_DATA_TOPIC} -H "Authorization: Bearer ${TOKEN}" | jq '.topic' -r)
```

Comment:
It is good practice to involve the evaluation of variables with double quotes to avoid unwanted behavior:
`"${DATA_BROKER_ADDRESS}/topic/${DEVICE_DATA_TOPIC}"`
This way there will be no risk of variables being expanded in multiple parameters and avoiding glob patterns